sequencer/src/api.rs

```diff
@@ -524,11 +528,31 @@ pub mod test_helpers {
             bind_version: V,
         ) -> Self {
             let mut cfg = cfg;
-            let (builder_task, builder_url) =
-                run_test_builder::<{ NUM_NODES }>(cfg.network_config.builder_port()).await;
+            let mut builder_tasks = Vec::new();
+            let mut legacy_builder_url = "http://example.com".parse().unwrap();
+            let mut marketplace_builder_url = "http://example.com".parse().unwrap();
+
+            if <V as Versions>::Base::VERSION <= MarketplaceVersion::VERSION {
```

Comment:
should this be just `<`, since we want to get marketplace builder when version `==` marketplace_version.